--- libsndfile-1.0.25/M4/octave.m4.octave~	2012-03-11 18:48:17.560254819 +0100
+++ libsndfile-1.0.25/M4/octave.m4	2012-03-11 18:48:29.876260956 +0100
@@ -119,7 +119,7 @@ if test "x$prog_concat" = "xyesyesyes" ;
 			2.*)
 				AC_MSG_WARN([Octave version 2.X is not supported.])
 				;;
-			3.0.*)
+			3.*)
 				OCTAVE_DEST_ODIR=`$OCTAVE_CONFIG --oct-site-dir | sed 's%^/usr%${prefix}%'`
 				OCTAVE_DEST_MDIR=`$OCTAVE_CONFIG --m-site-dir | sed 's%^/usr%${prefix}%'`
 
